npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/latest_version_finder.rb

```diff
@@ -59,10 +61,11 @@ def lowest_security_fix_version
           versions_array =
             if specified_dist_tag_requirement?
               [version_from_dist_tags].compact
-            else possible_versions
+            else possible_versions(false)
```

Comment:
Maybe clearer if this was a named arg:

```suggestion
            else possible_versions(filter_ignored_versions: false)
```